lessons/02_spacetime/02_01_1DConvection.ipynb

```diff
@@ -343,7 +343,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Remember that Python can also next commands, we could have instead written\n",
+    "Remember that we could have also combine the commands and write instead\n",
```

Comment:
Can you change this to read:
"Remember that Python can also combine commands, we could have instead written..."